### PR TITLE
Modify the terms by document style

### PR DESCRIPTION
**Pull request by liubin**
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/website/7026)

### CODE REVIEW

File: docs/concepts/services-networking/service.md [hunk 1]
```diff
@@ -124,8 +124,8 @@ Accessing a `Service` without a selector works the same as if it had a selector.
 The traffic will be routed to endpoints defined by the user (`1.2.3.4:9376` in
 this example).
 
-An ExternalName service is a special case of service that does not have
-selectors. It does not define any ports or endpoints. Rather, it serves as a
+An `ExternalName` service is a special case of service that does not have
+selectors. It does not define any ports or `Endpoints`. Rather, it serves as a
```
Comment (zacharysarah) [inline docs/concepts/services-networking/service.md]: No backticks needed for Endpoints.

File: docs/concepts/services-networking/service.md [hunk 2]
```diff
@@ -196,24 +196,24 @@ Note that in the above diagram, `clusterIP` is shown as `ServiceIP`.
 
 {% assign for_k8s_version="v1.9" %}{% include feature-state-beta.md %}
 
-In this mode, kube-proxy watches Kubernetes `services` and `endpoints`,
+In this mode, kube-proxy watches Kubernetes `Services` and `Endpoints`,
```
Comment (zacharysarah) [inline docs/concepts/services-networking/service.md]: No backticks needed here.

File: docs/concepts/services-networking/service.md [hunk 3]
```diff
@@ -196,24 +196,24 @@ Note that in the above diagram, `clusterIP` is shown as `ServiceIP`.
...
 calls `netlink` interface to create ipvs rules accordingly and syncs ipvs rules with Kubernetes
-`services` and `endpoints`  periodically, to make sure ipvs status is
-consistent with the expectation. When `service` is accessed, traffic will
-be redirected to one of the backend `pod`s.
+`Services` and `Endpoints`  periodically, to make sure ipvs status is
```
Comment (zacharysarah) [inline docs/concepts/services-networking/service.md]: No backticks needed here.

File: docs/concepts/services-networking/service.md [hunk 4]
```diff
@@ -196,24 +196,24 @@ Note that in the above diagram, `clusterIP` is shown as `ServiceIP`.
...
+consistent with the expectation. When `Service` is accessed, traffic will
```
Comment (zacharysarah) [inline docs/concepts/services-networking/service.md]: No backticks needed here.

File: docs/concepts/services-networking/service.md [hunk 5]
```diff
@@ -196,24 +196,24 @@ Note that in the above diagram, `clusterIP` is shown as `ServiceIP`.
...
+be redirected to one of the backend `Pod`s.
```
Comment (zacharysarah) [inline docs/concepts/services-networking/service.md]: No backticks needed here.

File: docs/concepts/services-networking/service.md [hunk 6]
```diff
@@ -224,11 +224,11 @@ it's not installed kube-proxy will fall back to iptables proxy mode.
 
 In any of proxy model, any traffic bound for the Service’s IP:Port is 
 proxied to an appropriate backend without the clients knowing anything 
-about Kubernetes or Services or Pods. Client-IP based session affinity 
-can be selected by setting service.spec.sessionAffinity to "ClientIP" 
+about Kubernetes or `Services` or `Pods`. Client-IP based session affinity
```
Comment (zacharysarah) [inline docs/concepts/services-networking/service.md]: No backticks needed here.

File: docs/concepts/services-networking/service.md [hunk 7]
```diff
@@ -373,7 +373,7 @@ either:
 ## Publishing services - service types
 
 For some parts of your application (e.g. frontends) you may want to expose a
-Service onto an external (outside of your cluster) IP address.
+`Service` onto an external (outside of your cluster) IP address.
```
Comment (zacharysarah) [inline docs/concepts/services-networking/service.md]: No backticks needed here.

File: docs/concepts/services-networking/service.md [hunk 8]
```diff
@@ -626,7 +626,7 @@ traffic. Nodes without any pods for a particular LoadBalancer service will fail
 the NLB Target Group's health check on the auto-assigned
 `spec.healthCheckNodePort` and not recieve any traffic.
 
-In order to achieve even traffic, either use a DaemonSet, or specify a
+In order to achieve even traffic, either use a `DaemonSet`, or specify a
```
Comment (zacharysarah) [inline docs/concepts/services-networking/service.md]: No backticks needed here.

File: docs/concepts/services-networking/service.md [hunk 9]
```diff
@@ -690,7 +690,7 @@ spec:
 ## Shortcomings
 
 Using the userspace proxy for VIPs will work at small to medium scale, but will
-not scale to very large clusters with thousands of Services.  See [the original
+not scale to very large clusters with thousands of `Services`.  See [the original
```
Comment (zacharysarah) [inline docs/concepts/services-networking/service.md]: No backticks needed here.

File: docs/concepts/services-networking/service.md [hunk 10]
```diff
@@ -794,7 +794,7 @@ through a load-balancer, though in those cases the client IP does get altered.
 
 #### Ipvs
 
-Iptables operations slow down dramatically in large scale cluster e.g 10,000 Services. IPVS is designed for load balancing and based on in-kernel hash tables. So we can achieve performance consistency in large number of services from IPVS-based kube-proxy. Meanwhile, IPVS-based kube-proxy has more sophisticated load balancing algorithms (least conns, locality, weighted, persistence).
+Iptables operations slow down dramatically in large scale cluster e.g 10,000 `Services`. IPVS is designed for load balancing and based on in-kernel hash tables. So we can achieve performance consistency in large number of services from IPVS-based kube-proxy. Meanwhile, IPVS-based kube-proxy has more sophisticated load balancing algorithms (least conns, locality, weighted, persistence).
```
Comment (zacharysarah) [inline docs/concepts/services-networking/service.md]: No backticks needed here.

**Comment by zacharysarah** [comment]
@liubin 👋 Please resolve the merge conflicts.

**Comment by liubin** [comment]
Hi @zacharysarah , thank your review and I had edited it.

**Pull request merged**